### DISCUSSION TITLE
Feature Request: Disable the Installed Updates Control Panel redirection to the Settings app

### DISCUSSION THREAD

**Discussion by sdw4938** [valinet/ExplorerPatcher#647] [Ideas]
Here is the Vive command syntax: ".\vivetool addconfig 36731482 1"

Velocity number thing: "36731482"

This doesn't yet happen in the Release version but will be implemented in the future.

Currently the redirection happens in builds 22523+

**Comment by valinet** [comment]
Yeah, someone else or me will have to look into it and determine some way similar to what we have to do this. Velocities are nice to know, but useless long term as they use them temporarily just for A/B testing. After they disregard any feedback from the community and decide on the worse alternative, they disable the old code path altogether in code and that’s it.
